### PR TITLE
Alert polish and brushup

### PR DESCRIPTION
**Pull request by DmitriyRubetskoy**
Changes:

-Removed rough pixel edges around alerts

-Drew an original alert icon for being pulled

-Made a separate stun alert folder (for proper structure, the alert icon is still the same)

-Made minor changes to descriptions of the alerts making them either more informational or less of a mouthful

-An extra stun icon which might be used later, but I can delete it from the commit if a redundant png file is not a good idea

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
Attach pictures of what they look like to the PR.

File: Resources/Prototypes/Alerts/alerts.yml [hunk 1]
```diff
@@ -69,22 +69,19 @@
   icon: /Textures/Interface/Alerts/Weightless/weightless.png
   name: Weightless
   description: >-
-    Gravity has ceased affecting you, and you're floating around aimlessly. You'll need something large and heavy, like a
-    wall or lattice, to push yourself off if you want to move. A jetpack would enable free range of motion. A pair of
-    magboots would let you walk around normally on the floor. Barring those, you can throw things, use a fire extinguisher,
-    or shoot a gun to move around via Newton's 3rd Law of Motion.
+    Gravity has ceased affecting you, and you're floating around aimlessly. Find something sturdy to hold onto, or expand force in a direction opposite of you. Magnetic boots or jetpacks will help you move with more control
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Alerts/alerts.yml:72]: "or expand force in a direction opposite from you." -> Not good. Needs to be rewritten.

"Magnetic boots or jetpacks will" -> "Mag-boots or a jetpack would"

File: Resources/Prototypes/Alerts/alerts.yml [hunk 2]
```diff
@@ -69,22 +69,19 @@
...
 
 - type: alert
   alertType: Stun
-  icon: /Textures/Objects/Weapons/Melee/stunbaton.rsi/stunbaton_off.png
+  icon: /Textures/Interface/Alerts/Stun/Stunned.png #Should probably add a proper Icon
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Alerts/alerts.yml:76]: I'm assuming you copy/pasted the stunbaton into the Alerts folder. Don't do that, it was fine before.

File: Resources/Prototypes/Alerts/alerts.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@
   icon: /Textures/Interface/Alerts/Pressure/highpressure.png
   maxSeverity: 2
   name: "[color=red]High Pressure[/color]"
-  description: "The air around you is [color=red]hazardously thick[/color]. A [color=green]fire suit[/color] would protect you."
+  description: "The air around you is [color=red]hazardously thick[/color]. A [color=green]fire suit[/color] would be enough protect you."
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Alerts/alerts.yml:42]: > "A pressurized suit would be enough to protect you."

**Review by SweptWasTaken — CHANGES_REQUESTED** [review]
Left some reviews. I don't know how I feel about you putting an icon in there even though we told you we'd probably never use it.

File: Resources/Prototypes/Alerts/alerts.yml (hunk 3 above)
Comment (DmitriyRubetskoy) [reply Resources/Prototypes/Alerts/alerts.yml:42]: Difference is that at this point a firesuit would be enough to protect you, the harsher atmosphere conditions have a "Space suit"

**Comment by DmitriyRubetskoy** [comment]
Oh god oh fuck wait

**Comment by DmitriyRubetskoy** [comment]
image
image
Don't know how to debug being pulled

**Comment by SweptWasTaken** [comment]
> Don't know how to debug being pulled

Spawn
Respawn
Pull your old character
Right click old character
Debug
Control mob

**Comment by DmitriyRubetskoy** [comment]
image
Seems good to me

**Comment by DmitriyRubetskoy** [comment]
image
Decided to clean it up somewhat, it had extra pixels

**Comment by SweptWasTaken** [comment]
The only difference I can tell is that the after appears to have slightly higher contrast meaning the fine details are harder to make out. I'll leave this up to others to judge though.

**Comment by SweptWasTaken** [comment]
My bad, pressed the wrong button.

**Comment by AJCM-git** [comment]
The only one that i think looks better/different is the no gravity one, maybe update the PR so it only includes that one?

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**